README.md

```diff
@@ -1,2 +1,335 @@
+by toggling its `open=""` attribute to true.
+The `options` include:
+    - `duration`: how long to show the toast, in milliseconds. Defaults to ???
+    - `multiple`: ???
+    - `newestOnTop`: ???
+
+TODO([#19](https://github.com/jackbsteinberg/std-toast/issues/19)): how do `multiple` and `newestOnTop` work?
+
+- `.hide()`: hides the toast element,
+by toggling its `open=""` attribute to false.
+- `.toggle([state])`: toggles the toast element,
+by hiding it if it's being shown and showing it if it's being hidden,
+or changing the `open=""` attribute to `state` if given.
+
+#### Events
+
+A `<std-toast>` element can fire the following events:
+
+- `"show"`: the toast was shown
+- `"hide"`: the toast was hiddne, either explicitly by the user, or via the timeout.
+  (Note: if animations were applied, the toast may not be entirely invisible at the time this event fires)
+    - TODO: should we consider separate events for the start and end of any hide animation?
+      This seems hard to do correctly if the user customizes the animation, though.
+- `"actionclick"`: the toast's call-to-action button or link was clicked, if one exists.
+    - TODO([#11](https://github.com/jackbsteinberg/std-toast/issues/11)): flesh out exactly how the action works and how this event is linked.
+
+### `showToast(message, options)`
+
+The `"std:elements/toast"` module also exports a convenience function,
+`showToast()`,
+which allows creating and showing toasts entirely from JavaScript.
+Behind the scenes,
+`showToast()` creates a `<std-toast>`,
+sets it up using the given message and options,
+inserts it as the last child of `<body>`,
+and then adds the `open=""` attribute,
+to make the toast visible.
+Finally,
+it returns the created `<std-toast>` element,
+allowing further manipulation by script.
+
+`message` is a string that will be inserted as a text node
+(TODO: or as a `<p>` element?) into the created `<std-toast>`.
+
+`options` allows configuring both the attributes of the `<std-toast>`,
+and the options for this particular showing of the toast.
+Thus, the possible options are:
+
+- `theme`, like the attribute
+- `position`, like the attribute
+- `closeButton`, like the attribute
+- `duration`, like the `show()` option
+- `multiple`, like the `show()` option
+- `newestOnTop`, like the `show()` option
+
+TODO([#11](https://github.com/jackbsteinberg/std-toast/issues/11)): How should actions be handled?
+
+### Default styles
+
+The standard toast will come with these default styles,
+which developers will be able to change to customize look and feel.
+
+#### `std-toast:not([open])`
+
+- `display: none`: this will keep the toast hidden from view and searching
+when the `open` attribute is not present.
+
+#### `std-toast([open])`
+
+TODO: figure out some default styles, and state them here.
+Additionally figure out default styles for themes.
+
+### Appearance customization
+
+TODO: explain any API for appearance customization,
```

Comment:
I think we should have a tracking issue for this. Similar to https://github.com/tkent-google/std-switch/issues/7, it just gives us a place to put thoughts as we develop the prototype and get an idea of what we're doing.

Reply:
Addressed in #20